javascript/javascript1/week1/homework.js

```diff
@@ -0,0 +1,91 @@
+// Step 2
+// My freecodecamp username - @Natata08
+
+//Step 3
+// Age-ify (A future age calculator)
+
+const yearOfBirth = 1986; //number
+const yearFuture = 2025; //number
+const age = yearFuture - yearOfBirth;
+const messageAboutFututeAge = 'You will be ' + age + ' years old in ' + yearFuture;
```

Comment:
`Futute` is a typo, it seems you would like to say `Future`

Reply:
I fixed it. Thank you so much. Code Spell Checker helped me find another typo.